eventmesh-common/src/main/java/org/apache/eventmesh/common/config/ConfigurationWrapper.java

```diff
@@ -80,7 +80,7 @@ public int getIntProp(String configKey, int defaultValue) {
         if (StringUtils.isEmpty(configValue)) {
             return defaultValue;
         }
-        Preconditions.checkState(StringUtils.isNumeric(configKey), String.format("%s error", configKey));
+        Preconditions.checkState(StringUtils.isNumeric(configValue), String.format("%s error", configKey));
```

Comment:
here String.format is still configKey?

Reply:
Yes, do we need to use configValue?
or use `String.format("key:%s value:%s error", configKey, configValue)`

Reply:
I prefer `String.format("key:%s value:%s error", configKey, configValue)` this.

Reply:
Done